src/urlscanio/utils.py

```diff
@@ -66,6 +66,17 @@ def create_arg_parser():
         ),
         type=str
     )
+    group.add_argument(
+        "-q", "--search-query",
+        help="Submit a search request for the given query.",
+        type=str
+    )
+    group.add_argument(
```

Comment:
I dig the search command, but I want to think a bit more about the `-g/--get-result` flag. Feels like it might get confused with `-r`.

Just to make it clear, I like the functionality, just want to make sure we introduce it in the best way possible. Could be a different flag name, could be combining it with the `-r` flag somehow. Will think it through a bit more, but happy to hear your thoughts.